frontends/ol-components/src/components/Card/Card.tsx

```diff
@@ -196,54 +267,64 @@ const Card: Card = ({ children, className, size, href }) => {
     else if (child.type === Actions) actions = child.props
   })
 
+  const hasHref = typeof href === "string"
+  const handleHrefClick = useClickChildHref(href, onClick)
+  const handleClick = hasHref && forwardClicksToLink ? handleHrefClick : onClick
+
   const allClassNames = ["MitCard-root", className ?? ""].join(" ")
 
   if (content) {
     return (
-      <Wrapper className={allClassNames} size={size}>
-        <_Container className={className} href={href!}>
-          {content}
-        </_Container>
-      </Wrapper>
+      <Container
+        {...others}
+        className={allClassNames}
+        size={size}
+        onClick={handleClick}
+      >
+        {content}
+      </Container>
     )
   }
 
   return (
-    <Wrapper className={allClassNames} size={size}>
-      <_Container href={href!} scroll={!href?.startsWith("?")}>
-        {image && (
-          // alt text will be checked on Card.Image
-          // eslint-disable-next-line styled-components-a11y/alt-text
-          <Image
-            className="MitCard-image"
-            size={size}
-            height={170}
-            width={298}
-            {...(image as ImageProps)}
-          />
-        )}
-        <Body>
-          {info.children && (
-            <Info className="MitCard-info" size={size} {...info}>
-              {info.children}
-            </Info>
-          )}
-          <Title className="MitCard-title" size={size} {...title}>
-            {title.children}
-          </Title>
-        </Body>
-        <Bottom>
-          <Footer className="MitCard-footer" {...footer}>
-            {footer.children}
-          </Footer>
-        </Bottom>
-      </_Container>
-      {actions.children && (
-        <Actions className="MitCard-actions" {...actions}>
-          {actions.children}
-        </Actions>
+    <Container
+      {...others}
+      className={allClassNames}
+      size={size}
+      onClick={handleClick}
+    >
+      {image && (
+        // alt text will be checked on Card.Image
+        // eslint-disable-next-line styled-components-a11y/alt-text
+        <Image
+          className="MitCard-image"
```

Comment:
Not one for this PR as this hasn't changed here, but can we use `data-testid`s if these are just for selection in the tests (or ideally avoid altogether). `.MitCard-root` and `.MitCard-info` are the only ones in use for CSS selectors.

Reply:
We do use `MitCard-root` and `MitCard-info` for styling at least once, which is why they were added. When I added them, I added for other parts (like `MitCard-title`) that made sense but weren't actually used (as of yet).